lk2nd/display/panel/generated/lk_panel_innolux_qhd_video.h

```diff
@@ -0,0 +1,380 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2025 FIXME
+// Generated with linux-mdss-dsi-panel-driver-generator from vendor device tree:
+//   Copyright (c) 2014, The Linux Foundation. All rights reserved. (FIXME)
```

Comment:
Can you add the additional make arguments for panel driver to the comment inside the dts, like some other devices do? So the way you build it is written down somewhere.

Reply:
what about mentioning the commit on `Documentation/devices.md`?

Reply:
Done, let me know if there's something to improve :D.